If I can shave off 7 seconds off every exp-run this patch seems totally worth it
--- configure.orig	2014-12-11 13:46:28 UTC
+++ configure
@@ -5343,18 +5343,12 @@ fi
 
 
 echo $ECHO_N "ordering CD from http://tobi.oetiker.ch/wish $ECHO_C" 1>&6
-sleep 1
 echo $ECHO_N ".$ECHO_C" 1>&6
-sleep 1
 echo $ECHO_N ".$ECHO_C" 1>&6
-sleep 1
 echo $ECHO_N ".$ECHO_C" 1>&6
-sleep 1
 echo $ECHO_N ".$ECHO_C" 1>&6
-sleep 1
 { $as_echo "$as_me:${as_lineno-$LINENO}: result:  just kidding ;-)" >&5
 $as_echo " just kidding ;-)" >&6; }
-sleep 2
 echo
 echo "----------------------------------------------------------------"
 echo "Config is DONE!"
